### ISSUE TITLE
Theming

### ISSUE THREAD

**Issue by janhancic** [JedWatson/touchstonejs#4]
Will this support theming? Right now it looks iOSy but I wouldn't want to use this theme on android for instance.

**Comment by colinramsay** [comment]
Is the idea that you'll just be able to override the LESS variables to produce your own theme?

**Comment by JedWatson** [comment]
@colinramsay yes, that's the plan. I expect we'll also ship with a few default themes that you can select from, including iOS-y and Android-y versions.

We're also looking into alternatives for how best to implement theming with React, including potentially specifying styles in Javascript rather than LESS. Ultimately the decision will depend on how to most simply achieve the best flexibility, and how different JS-driven DOM structure or UI behaviours may need to be to support different themes (i.e. if you want to control the behaviour of UI elements as part of the theme, modifying the CSS may not be sufficient, and it may be simpler to control everything though a JS-driven themeing API)

**Comment by arnold-almeida** [comment]
refs #52

**Comment by TrySpace** [comment]
Hold your android horses: http://material-ui.com

Also very much a work in progress. But a perfect match if you ask me.
